packages/core/test/util/transactionSummaryInspector.test.ts

```diff
@@ -1017,4 +1043,81 @@ describe('Transaction Summary Inspector', () => {
       }
     });
   });
+
+  it('displays unresolved inputs on timeout', async () => {
+    const resolveDelay = 5;
```

Comment:
No a required change, just a nitpick for slightly better test readability
```suggestion
    const resolveDelay = Milliseconds(5);
```

Reply:
Fixed by 87baffa8fd705826f0073bec5d318204a87344e5